docs/content/en/docs/clustermgmt/observability/expose-metrics.md

```diff
@@ -0,0 +1,146 @@
+      haproxy.cfg: |
+        defaults
+          mode http
+          timeout connect 5000ms
+          timeout client 5000ms
+          timeout server 5000ms
+          default-server maxconn 10
+
+        frontend kube-proxy
+          bind ${NODE_IP}:10249
+          http-request deny if !{ path /metrics }
+          default_backend kube-proxy
+        backend kube-proxy
+          server kube-proxy 127.0.0.1:10249 check
+
+        frontend kube-controller-manager
+          bind ${NODE_IP}:10257
+          http-request deny if !{ path /metrics }
+          default_backend kube-controller-manager
+        backend kube-controller-manager
+          server kube-controller-manager 127.0.0.1:10257 ssl verify none check
+
+        frontend kube-scheduler
+          bind ${NODE_IP}:10259
+          http-request deny if !{ path /metrics }
+          default_backend kube-scheduler
+        backend kube-scheduler
+          server kube-scheduler 127.0.0.1:10259 ssl verify none check
+    EOF
+    ```
+
+4.  Create a daemonset for the proxy and mount the config map volume onto the proxy pods.
+    
+    Below is an example configuration for the HAProxy daemonset.
+    ```bash
+    cat << EOF | kubectl apply -f -
+    apiVersion: apps/v1
+    kind: DaemonSet
+    metadata:
+      name: metrics-proxy
+    spec:
+      selector:
+        matchLabels:
+          app: metrics-proxy
+      template:
+        metadata:
+          labels:
+            app: metrics-proxy
+        spec:
+          tolerations:
+          - key: node-role.kubernetes.io/control-plane
+            operator: Exists
+            effect: NoSchedule
+          hostNetwork: true
+          containers:
+            - name: haproxy
+              image: public.ecr.aws/eks-anywhere/kubernetes-sigs/kind/haproxy:v0.20.0-eks-a-54
+              env:
+                - name: NODE_IP
+                  valueFrom:
+                    fieldRef:
+                      apiVersion: v1
+                      fieldPath: status.hostIP
+                - name: TOKEN
+                  value: "$(cat /var/run/secrets/kubernetes.io/serviceaccount/token)"
+              ports:
+                - name: kube-proxy
+                  containerPort: 10249
+                - name: kube-ctrl-mgr
+                  containerPort: 10257
+                - name: kube-scheduler
+                  containerPort: 10259
+              volumeMounts:
+                - mountPath: "/usr/local/etc/haproxy"
+                  name: haproxy-config
+          volumes:
+            - configMap:
+                name: metrics-proxy
+              name: haproxy-config
+    EOF
+    ```
+
+5.  Verify that the metrics are exposed to the client pods by running the following commands:
+    ```bash
+    kubectl exec -it {client-pod-name} -n {client-pod-namespace} -- bash
+    export TOKEN=$(cat /var/run/secrets/kubernetes.io/serviceaccount/token)
+    curl -H "Authorization: Bearer ${TOKEN}" http://{node-IP}:{component-port}/metrics
```

Comment:
nit: it would be better if you gave an example that worked out of the box, using an ephemeral container with `kubectl debug` or simply a `kubectl run`.

Reply:
I have updated the docs to have the client permissions configuration steps in a separate subsection. I think it should be fine to continue with the current example in the context of that subsection.